File: src/demunity.lib/Data/Models/PhotoComment.cs
using System;

namespace demunity.lib.Data.Models
{
    public class PhotoComment
    {
        public PhotoId PhotoId { get; set; }
        public UserId UserId { get; set; }
        public string UserName { get; set; }
        public string Text { get; set; }
        public DateTimeOffset CreatedTime { get; set; }

    }
}

File: src/demunity/typescript/photocomment.ts
class PhotoCommentObject {
	photoId: string;
	userId: string;
	userName: string;
	time: Date;
	text: string;

	constructor(photoId: string, userId: string, userName: string, time: string, text: string) {
		this.photoId = photoId;
		this.userId = userId;
		this.userName = userName;
		this.time = new Date(Date.parse(time));
		this.text = text;
	}
}